courses/free-courses-te.md

```diff
@@ -37,3 +126,28 @@
 
 * [R Programming Language Training Videos In telugu - R ట్రైనింగ్ వీడియోస్ ఇన్ తెలుగు R programming ట్రైనింగ్ వీడియోస్ ఇన్ తెలుగు](https://www.youtube.com/playlist?list=PLXx2-0oYp1LO9H8ciGQaTr6SN80dteTlc) - VLR Training (YouTube)
 
+
+### ReactJS
+
+* [React Js In Telugu](https://www.youtube.com/playlist?list=PLBOdaohjTONhaVqygz5880g1vW2paRZw_) - srinivasarao aleti
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.
use java script